docs/networking/best-practice/multiple-nics-vlan-aware-switch.md

```diff
@@ -0,0 +1,96 @@
+---
+sidebar_position: 50
+keywords:
+  - Harvester
+  - harvester
+  - Rancher
+  - rancher
+  - Network
+  - network
+  - VLAN
+  - vlan
+Description: Harvester is built on top of Kubernetes, and uses the [CNI](https://github.com/containernetworking/cni) as the interface between network providers and Kubernetes pod networking. Naturally, we implement the Harvester network based on CNI. Moreover, the Harvester UI integrates the network configuration in order to provide a user-friendly way to configure networks for VMs.
+---
+
+# Mulitple NICs with VLAN-aware Switch
+
+In this best practice guide on how to configure "VLAN-aware", we will introduce Harvester VLAN network and external switch configuration for common scenario.
+
+## Architecture
+
+Hardware:
+
+- Three Harvester servers with daul ports network card.
+- One or more VLAN-aware switch(es). We will use "Cisco like" configuration as example.
+
+Network Specification:
+
+- Assume that the subnet of the Harvester hosts is in VLAN 100.
+- Assume that the VMs are in the VLAN 101-200.
+
+Cabling:
+
+- The Harvester servers are connected to the switch in a port from `1` to `6`.
+
+The following diagram illustrates the cabling used for this guide:
+
+   ![mulitple-nics-vlan-aware.png](assets/mulitple-nics-vlan-aware.png)
+
+## External Switch Configuration
+
+For the external switch configuration, we'll use a "Cisco like" configuration as example. You can apply the following configurations to your switch:
+
+
+For `harvester-mgmt` ports:
+```
+switch# config terminal
+switch(config)# interface ethernet1/<Port Number>
+switch(config-if)# switchport
+switch(config-if)# switchport mode access
+switch(config-if)# switchport access 100
+switch(config-if)# no shutdown
+switch(config-if)# end
+switch# copy running-config startup-config
+```
+
+!!! note
+    In this case, you need to avoid using `harvester-mgmt` as VLAN Network interface. This settings will only allow the traffic in the same subnet of `harvester-mgmt` and disallow other VLAN traffic.
```

Comment:
```suggestion
    In this case, you need to avoid using `harvester-mgmt` as the VLAN Network interface. This setting will only allow the traffic in the same subnet of `harvester-mgmt` and disallow other VLAN traffic.
```